tools/mk/Makefile.node_prebuilt.defs

```diff
@@ -122,7 +123,7 @@ else
 	NODE_PREBUILT_NAME := $(NODE_PREBUILT_VERSION)-$(NODE_PREBUILT_TAG)-$(NODE_PREBUILT_IMAGE)
 endif
 NODE_PREBUILT_PATTERN := sdcnode-$(NODE_PREBUILT_NAME)-$(NODE_PREBUILT_BRANCH)-.*\.tgz
-NODE_PREBUILT_DIR ?= https://download.joyent.com/pub/build/sdcnode/$(NODE_PREBUILT_IMAGE)/$(NODE_PREBUILT_BRANCH)-latest/sdcnode/
+NODE_PREBUILT_DIR ?= https://download.tritondatacenter.com/pub/build/sdcnode/$(NODE_PREBUILT_IMAGE)/$(NODE_PREBUILT_BRANCH)-latest/sdcnode/
```

Comment:
I think we can change this to `https://us-central.manta.mnx.io/Joyent_Dev/public/releng/sdcnode/$(NODE_PREBUILT_IMAGE)/$(NODE_PREBUILT_BRANCH)-latest/sdcnode/`

Reply:
Seems a good idea.

Reply:
Actually it's not gonna work because `$(NODE_PREBUILT_BRANCH)-latest` doesn't work anymore becuase of MantaV2.  Sticking with what I got.

Reply:
We've switched to engbld to fix this.